content/07.pathogenesis.md

```diff
@@ -46,22 +46,30 @@ One severe example includes reports of cytokine storm-like responses in patients
 
 #### Reproduction Number and Dynamics of Transmission
 
-Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
-The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@isbn:9780691116174].
-The basic reproduction number, R~0~, is the expected number of new (secondary) infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
-R~0~ is a unitless number which in mechanistic models is a combination of parameters principally related to the transmission rate (rate of infection transmitting interactions for an infectious individual) and the infectious period [@doi:10.1126/science.abb5659; @isbn:9780691116174].
-A pathogen can invade a susceptible population only if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996].
-The effective reproduction number, R~t~, describes how the reproduction number may change over time, and is used to quantify deviations in R from R~0~ that would occur, for example, as some fraction of the population became infected or as interventions were put into place.
+Disease spread dynamics can be estimated using R~0~, the basic reproduction number, and R~t~, the effective reproduction number. 
+Thus, accurate estimates are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
+R~0~ is one of most critical epidemiological parameters [@isbn:9780691116174]; R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x]. 
+A simple mechanistic model used to describe infectious disease dynamics is a susceptible-infected-recovered (SIR) state-transition model. 
```

Comment:
Question/clarification for @LucyMcGowan and please excuse if this is nonsense because I'm not great at jargon & I'm not taking the time to think this through carefully, so this is just me double-checking. Would you double-check that "state-transition" is a term general or specific enough to apply here. I can guess what you mean, but I wanted to double-check that nothing is implied that's too specific or general.
-basic SIR model is usually differential equations model (deterministic, ~continuous variables corresponding to S & I & R, ~continuous time) which in the infectious disease modeling literature is called "compartmental model"
If I go through it with pencil and paper, I can derive relations based on assumptions, limits, and probability theories but I'm not gonna do that right now, so I just wanted to double-check that the term you used applies to the description I put here for basic SIR model. (your terms reminded me of Markov models but again I'd have to go through pencil-and-paper to remind myself of the forms I learned & I know there are also variations.)
-Sometimes people will use SIR as a schematic to refer to other types of models which might have mathematically different features/structures/assumptions or they may just use the term SIR to refer to the conceptual idea of categorizing people as S or I or R without specifying the type of model. But I'm wondering if this relates to how "state-transition model" can/would most likely be interpreted? (It might depend on which fields you read the most, not sure if what I'm writing is nonsense because i don't follow jargon myself but I'm just double-checking)

Reply:
Hmm, in my group we do refer to these as state-transition models (ie the states: S, I, R, the transitions $\beta$, $\gamma$) BUT I come from a biostatistics background, so it's possible that it is really not common elsewhere. I'll change it to compartmental model :+1: